share/completions/mariner.fish

```diff
@@ -0,0 +1,35 @@
+set -l subcommands complete config details download help magnet open search
+
+# Subcommands
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'complete' --description "Print bash completion command (cliff)"
```

Comment:
What does that "cliff" mean here?

Reply:
It's just a comment that comes from the underlying CLI framework. I've removed it.